hackclub.com.yaml

```
@@ -2018,6 +2019,10 @@ pic._domainkey.mail:
   ttl: 600
   type: TXT
   value: k=rsa
+  pinetree.hack.com:
+  - ttl: 600
+    type: A
+    value: 127.0.0.1.
```

couple things here:


```suggestion
pinetree:
  - ttl: 600
    type: A
    value: 127.0.0.1.
```

127.0.0.1 is the loopback address, what do you want this domain to point to?